ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/query/VaultNamedQueryTransformer.java

```diff
@@ -28,10 +27,10 @@
  * }
  * }</pre></li></ul>
  *
- * @param <T> Type of the state returned from the database.
- * @param <R> Type that the original state is transformed into.
+ * @param <T> Type of the data returned from the database.
```

Comment:
Update the Javadocs for this and the filter, just changing the `state` in the argument to `data` is enough for me now. Everything else should still hold.